app/src/main/java/com/github/onlynotesswent/ui/common/Popups.kt

```diff
@@ -137,3 +137,49 @@ fun CreationDialog(
             }
       })
 }
+
+@Composable
+fun EnterTextPopup(
+    onDismiss: () -> Unit,
+    onConfirm: (String) -> Unit,
+    action: String,
+    type: String
+) {
+
+  var text by remember { mutableStateOf("") }
+
+  AlertDialog(
+      onDismissRequest = onDismiss,
+      title = { Text("$action $type") },
+      text = {
+        Column(
+            modifier = Modifier.padding(16.dp).testTag("${type}Dialog"),
+            verticalArrangement = Arrangement.spacedBy(8.dp),
+            horizontalAlignment = Alignment.CenterHorizontally) {
+              OutlinedTextField(
+                  value = text,
+                  onValueChange = { text = Folder.formatName(it) },
```

Comment:
You shouldn't use folder's formatter but instead create your own `Notification.formatTextNotification(String)` with a custom limit, e.g. 100 chars.

Reply:
Yes you are right I will do that